packages/io/__tests__/IO.test.ts

```diff
@@ -52,16 +52,6 @@ describe("IO tests", () => {
         expect(IO.isDir("pretend/file")).toBe(false);
     });
 
-    it("should get an error for no input on normalizeExtension", () => {
```

Comment:
Instead of removing this test, how about modifying it to test the new behavior. The test name could change to `it('should return no extension')` or something like that.